### PR TITLE
refactor(redirector)!: update configuration and logging

### PR DESCRIPTION
**Pull request by madhavilosetty-intel**
[refactor(redirector)!: update configuration and logging](https://github.com/open-amt-cloud-toolkit/ui-toolkit/pull/787/commits/216d0a9ea941c130ca842965a955cc434132d288) 

BREAKING CHANGE: AMT Redirector requires a configuration object and ILogger interface deprecated.

## PR Checklist

- [ ] Unit Tests have been added for new changes
- [ ] API tests have been updated if applicable
- [ ] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with Apache 2.0 and clearly outlined the added dependency.

### CODE REVIEW

File: dist/core/logger.d.ts
Comment (rsdmike) [inline dist/core/logger.d.ts:1]: i thought you removed winston?

**Review by rsdmike — CHANGES_REQUESTED** [review]
almost there, in the commit message you mentioned that the ILogger is deprecated, but actually its removed.

File: dist/core/logger.d.ts
Comment (madhavilosetty-intel) [reply dist/core/logger.d.ts:1]: Winston wasn't there earlier. I added and removed.

File: dist/bundles/core.bundle.js
[the remark this answers is not in the document]
Comment (madhavilosetty-intel) [reply dist/bundles/core.bundle.js:1]: removed bundles

**Review by rsdmike — APPROVED** [review]

**Pull request merged**